src/main/kotlin/de/terrestris/geoserver/wps/DistinctValues.kt

```diff
@@ -117,11 +121,21 @@ class DistinctValues(private val geoServer: GeoServer) : GeoServerProcess {
                 }
                 LOGGER.fine("Custom SQL: $sql")
             }
+            if (order != null && (order.toLowerCase() == "asc" || order.toLowerCase() == "desc")) {
+                sql += " ORDER BY $propertyName $order"
+            }
+            if (limit != null) {
+                sql += " LIMIT $limit"
+            }
```

Comment:
Wouldn't it make sense to now log the SQL on finest level, and not some lines above. I am only looking at the diff, so might be wrong

Reply:
right, enhanced